src/components/Button/index.js

```diff
@@ -34,11 +34,17 @@ export default class Button extends PureComponent {
         background: ${({ primary }) =>
           primary ? theme.color.primaryLight : theme.color.greyLightest};
       }
+
+      :disabled {
+        pointer-events: none;
+        opacity: 0.7;
```

Comment:
can you also add `cursor: not-allowed` to the disabled styles?

Reply:
Np. Done.